wiki/docs/building.md

```diff
@@ -0,0 +1,26 @@
+Building
+---
+
+With toolchain installed you are ready to compile Mimiker. Before you run
+`make` in project's root directory, please have a look at `config.mk` file.
```

Comment:
Why should I have a look there? What is important in `config.mk`? Maybe elaborate on that.